### ISSUE TITLE
Color contrast bug on Do button - orange

### ISSUE THREAD

**Issue by joe-watkins** [joe-watkins/kfdb#10]
Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds

[more information Link opens in a new window](https://dequeuniversity.com/rules/axe/4.10/color-contrast?application=AxeChrome)
Element Location:
.bg-amber-600 > span
<span class="flex items-center justify-center gap-2">
To solve this problem, you need to fix the following:
Element has insufficient color contrast of 3.18 (foreground color: #ffffff, background color: #d97706, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

Related Node
<button type="button" class="w-full flex items-center justify-center px-3 py-2 text-sm font-semibold rounded-md focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-offset-gray-900/50 bg-amber-600 hover:bg-amber-500 text-white focus:ring-amber-400 disabled:opacity-50 disabled:cursor-wait transition-colors">

How to fix:
Choose a darker orange that is WCAG compliant. 4.5:1 

<img width="1426" height="652" alt="Image" src="https://github.com/user-attachments/assets/2272a024-90eb-402d-8130-e252f832303e" />
